<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="weld_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Weld Geometry:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="weld">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Edge Welded Beam</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TG9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tekken</string>
    </property>
   </item>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>781</width>
     <height>561</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="home">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>321</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>Please select a Weld Geometry....</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="TG9">
    <widget class="QLabel" name="b_label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Weld Pass Length (B)</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="b_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="a_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="c_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="a_label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>171</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Weld Pass Thickness (A)</string>
     </property>
    </widget>
    <widget class="QLabel" name="c_label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>151</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Weld Pass Height (C)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Geometry (mm):</string>
     </property>
    </widget>
    <widget class="QLabel" name="d_label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>171</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Base plate Thickness (D)</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="d_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="f_label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>151</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Base plate Height (F)</string>
     </property>
    </widget>
    <widget class="QLabel" name="e_label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>151</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Base plate Length (E)</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="e_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>200</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="f_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>240</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Meshing (mm):</string>
     </property>
    </widget>
    <widget class="QLabel" name="TG9_image">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>371</width>
       <height>241</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>TG9.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Edge_Weld">
    <widget class="QTextEdit" name="a">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="edge_weld_image">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>0</y>
       <width>421</width>
       <height>221</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>EdgeWelded.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="c_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>151</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Weld Pass Height (C)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Geometry (mm):</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="c">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="a_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>171</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Weld Pass Thickness (A)</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="b">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="b_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Weld Pass Length (B)</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="d">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="d_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>151</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Base plate Height (D)</string>
     </property>
    </widget>
    <widget class="QLabel" name="edge_weld_mesh">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>270</y>
       <width>531</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>EdgeWeldedMesh.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Meshing:</string>
     </property>
    </widget>
    <widget class="QLabel" name="k_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>430</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>K</string>
     </property>
    </widget>
    <widget class="QLabel" name="g_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>350</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>G</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="g">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="e">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="i">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="e_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>E</string>
     </property>
    </widget>
    <widget class="QLabel" name="i_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>390</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>I</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="k">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>420</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Element length (mm):</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="j">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>380</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="h">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>340</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="h_label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>350</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>H</string>
     </property>
    </widget>
    <widget class="QLabel" name="j_label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>390</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>J</string>
     </property>
    </widget>
    <widget class="QLabel" name="f_label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>310</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>F</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="f">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>300</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Refined Zone length (mm):</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="m">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>520</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="m_label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>530</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>K</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>530</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>J</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="l">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>520</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Tekken">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="button_ok">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>620</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_ok_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>620</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
